cmd/influx/main.go

```diff
@@ -425,16 +425,17 @@ func (c *CommandLine) parseInto(stmt string) string {
 }
 
 func (c *CommandLine) Insert(stmt string) error {
-	stmt = stmt[7:]
-	if strings.EqualFold(stmt[:4], "into") {
-		stmt = c.parseInto(stmt[5:])
+	i, point := parseNextIdentifier(stmt)
+	if !strings.EqualFold(i, "insert") {
+		fmt.Println("ERR: missing space after INSERT")
```

Comment:
The function `ParseCommand` that calls `Insert` has already determined that the line begins with "insert". `parseNextIdentifier` returns the first word in the string and the remainder. If the first word is not exactly "insert", a space is missing.